feel-engine/src/main/scala/org/camunda/feel/interpreter/FeelInterpreter.scala

```diff
@@ -420,6 +420,11 @@ class FeelInterpreter {
       case ValDayTimeDuration(y) => ValDateTime( x.minus(y) )
       case _ => error(y, s"expect Time, or Year-Month-/Day-Time-Duration but found '$x'")
     }
+    case ValDate(x) => y match {
+      case ValDate(y) => ValYearMonthDuration( Period.between(x, y) )
```

Comment:
The result should be "ValDayTimeDuration", similar to date-time. You must transform the date to a date-time with time = 0 and calculate the duration. Note that you need to swap the arguments in the `between` function.

Reply:
Of course - I need to turn the arguments around (I had copied my initial impl from ValTime in the subOp which just so happens to be the only subtraction that is `x, y` rather than `y, x`.

Could you expand on "transform the date to a date-time with time = 0" - is there an example of how that is done somewhere?

Reply:
Yep. Just call `date.atStartOfDay` which returns a date-time (thanks to Java 8 Date-Time API).